section6Arrays/reverseString/main.go

```diff
@@ -0,0 +1,10 @@
+package main
```

Comment:
no need to use package main. Main package should only be used for executables, meaning programs which you are going to run from command line, just like you were doing earlier.
Use package here like below
package reverseString

Also the file should be named as any name like revstr 
and the test file should named as revstr_test

Lets have a discussion whenever you have sometime. Ping me

Reply:
Fixed